si/orange3-imageanalytics.jaml

```diff
@@ -0,0 +1,497 @@
+            pca: false 
+        def `_get_assignments`:
+            sqeuclidean: false
+            lap: false
+import_images.py:
+    class `ImportImages`:
+        ImgData: false
+        path: false
+        format: false
+        height: false
+        width: false
+        size: false
+        ImgDataError: false
+        error: false
+        error_str: false
+        def `image_meta`:
+            *: false
+    def `scan`:
+        *: false
+        .*: false
+    def `image_meta_data`:
+        ascii: false
+        nan: false
+    def `create_table`:
+        category: kategorija
+        image name: ime slike
+        image: null
+        type: false
+        origin: false
+        size: velikost
+        width: širina
+        height: višina
+local_embedder.py:
+    class `LocalEmbedder`:
+        def `__init__`:
+            target_image_size: false
+            .ImageEmbedder.httpcache: false
+server_embedder.py:
+    class `ServerEmbedder`:
+        def `__init__`:
+            image/jpeg: false
+utils/embedder_utils.py:
+    class `ImageLoader`:
+        def `__init__`:
+            .ImageEmbedder.httpcache: false
+        def `load_image_or_none`:
+            RGB: false
+        def `load_image_bytes`:
+            JPEG: false
+        def `_load_image_from_url_or_local_path`:
+            http: false
+            https: false
+            Image skipped: Izpuščena slika
+            ftp: false
+            data: false
+utils/image_utils.py:
+    def `filter_image_attributes`:
+        type: false
+        image: false
+    def `extract_image_path`:
+        origin: false
+        http: false
+        https: false
+        ftp: false
+        data: false
+        /: false
+widgets/__init__.py:
+    Image Analytics: Analiza slik
+    Management and embedding of image data.: Upravljanje in vpetje slikovnih podatkov.
+    icons/Category-ImageAnalytics.svg: false
+    '#F7F5A8': false
+    {DEVELOP_ROOT}/doc/_build/html/index.html: false
+    {}/help/orange3-imageanalytics/index.html: false
+    data: false
+    http://orange3-imageanalytics.readthedocs.io/en/latest/: false
+widgets/owimageembedding.py:
+    def `run_embedding`:
+        squeezenet: false
+    class `OWImageEmbedding`:
+        Image Embedding: Vpetje slik
+        Image embedding through deep neural networks.: Vpetje slik z globokimi nevronskimi mrežami.
+        embedding: vpetje
+        image: slika
+        image embedding: vpetje slike
+        icons/ImageEmbedding.svg: false
+        class `Inputs`:
+            Images: Slike
+        class `Outputs`:
+            Embeddings: Vpetje
+            Skipped Images: Izpuščene slike
+        class `Warning`:
+            'No internet connection: switched to local embedder': 'Ni internetne povezave: preklopite na lokalni vgrajevalnik'
+            Please provide data with an image attribute.: Prosimo, navedite podatke z atributom slike.
+            {} images are skipped.: 'Število izpuščenih slik: {}'
```

Comment:
Spomnil sem se mogoče lepše oblike: `{} izpuščenih slik`. Pusti ali spremeni, po okusu.

Reply:
Je lepše, vendar če bo samo 1, bo izgledalo "šlampasto": 1 izpuščenih slik. Tukaj se pa ne da lepo sklanjati, ker je to string znotraj Msg().

Reply:
Prav imaš. Tole je res sitno. Obstaja en obvoz, ampak je zoprn in grd, tako da naj kar bo tako, kot si naredila.